operators/advanced_network/advanced_network/manager.cpp

```diff
@@ -192,6 +231,185 @@ Status Manager::allocate_memory_regions() {
   return Status::SUCCESS;
 }
 
+Status Manager::map_mrs() {
```

Comment:
could we call this `map_memory_regions` please?

As before, if you could explain what it does and in what context it's used here?